astro/src/components/get-started/DelegateStep3.astro

```diff
@@ -0,0 +1,20 @@
+---
+import Code from 'astro/components/Code.astro';
+
+---
+<dd>
+In your shell, type the following to start up the application:
+
+<Code code="cd app; npm install; npm run dev" />
```

Comment:
```suggestion
<Code code="cd app && npm install && npm run dev" />
```

This is short circuits things if there is an error.

Reply:
Check.